internal/resources/ibm/is_flow_log.go

```diff
@@ -0,0 +1,104 @@
+package ibm
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+// IsFlowLog struct represents <TODO: cloud service short description>.
+//
+// <TODO: Add any important information about the resource and links to the
+// pricing pages or documentation that might be useful to developers in the future, e.g:>
+//
+// Resource information: https://cloud.ibm.com/<PATH/TO/RESOURCE>/
+// Pricing information: https://cloud.ibm.com/<PATH/TO/PRICING>/
+type IsFlowLog struct {
+	Address       string
+	Region        string
+	TransmittedGB *int64 `infracost_usage:"transmitted_gb"`
+}
+
+// IsFlowLogUsageSchema defines a list which represents the usage schema of IsFlowLog.
+var IsFlowLogUsageSchema = []*schema.UsageItem{
+	{Key: "transmitted_gb", DefaultValue: 0, ValueType: schema.Float64},
+}
+
+// PopulateUsage parses the u schema.UsageData into the IsFlowLog.
+// It uses the `infracost_usage` struct tags to populate data into the IsFlowLog.
+func (r *IsFlowLog) PopulateUsage(u *schema.UsageData) {
+	resources.PopulateArgsWithUsage(r, u)
+}
+
+func (r *IsFlowLog) isFlowLogCostComponent() *schema.CostComponent {
+
+	freeRegions := []string{"us-east", "us-south", "eu-de", "eu-gb", "jp-tok"}
+
+	planType := "Paid"
+
+	if contains(freeRegions, r.Region) {
+		planType = "Free"
+	}
+
+	t := *r.TransmittedGB
+
+	startUsageAmount := "0"
```

Comment:
My previous PR solved the tiering for this model.
https://cloud.ibm.com/docs/sell?topic=sell-service-metering-integration#pricing